fcl/gen/text/textfilegen_HiggsScalar_M150_th2e-4_KDIF.fcl

```diff
@@ -0,0 +1,120 @@
+#     
+#     (-369.63,-214.61,-984.951) -- (-70.80,167.71,984.951) cm
+#     (  70.80,-214.61,-984.951) -- (369.63,167.71,984.951) cm
+#     
+# (which includes some cathode volume as well). Because of this deterministic
+# feature, `Geometry` service would not be required by this job. This
+# configuration demands it nevertheless for stronger consistency enforcement.
+# If this is not desired and the output files should be "geometry-agnostic",
+# just add in the overridden configuration file (see below for an example) the
+# following lines:
+# 
+#     services.Geometry:                    @erase
+#     services.GeometryConfigurationWriter: @erase
+#     services.ExptGeoHelperInterface:      @erase
+# 
+# The output file name will be `prodtext_standard_icarus_%tc-GenText.root`
+# (`%tc` being a timestamp assigned by art).
+# 
+# 
+# How to run a job with this configuration
+# =========================================
+# 
+# A different input file ("MyEvents.txt") can be processed by running a
+# configuration like the following:
+#     
+#     #include "prodtext_standard_icarus.fcl"
+#     
+#     physics.producers.generator.InputFileName: "MyEvents.txt"
+#     
+#     outputs.rootoutput.fileName: "MyEvents_icarus_%tc-%p.root"
+#     
+# where the last line changes the output file name for convenience (the same
+# effect can be achieved by specifying `-o "MyEvents_icarus_%tc-%p.root"`
+# argument to `lar` executable).
+# 
+# In every case, the number of events in the input file should be specified
+# either in the configuration (`source.MaxEvents: 10`) or on the command line
+# (`lar -n 10 ...`). Specifying a number of events larger than the ones in the
+# input files will result in the excess events being created with no particles.
+#
+#
+
+#include "messages_icarus.fcl"
+#include "geometry_icarus.fcl"
+#include "services_common_icarus.fcl"
+#include "rootoutput_icarus.fcl"
+
+
+# ------------------------------------------------------------------------------
+process_name: GenText
+
+
+# ------------------------------------------------------------------------------
+services: {
+
+  @table::icarus_art_services  # from `services_common_icarus.fcl`
+  
+  message: @local::icarus_message_services_interactive  # from `messages_icarus.fcl`
+  
+  @table::icarus_geometry_services  # from `geometry_icarus.fcl`
+  
+} # services
+
+
+# ------------------------------------------------------------------------------
+source: {
+  module_type:     EmptyEvent
+  timestampPlugin: { plugin_type: "GeneratedEventTimestamp" }
+}
+
+
+# ------------------------------------------------------------------------------
+physics: {
+
+  producers: {
+    
+    # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+    
+    generator: {
+      module_type:    TextFileGen
+      InputFileName: "/icarus/app/users/jdyer/scalargen/my_dev_area_placeTextFiles/srcs/icarus_data/icarus_data/HiggsPortal/150MeV_mySim_unweighted_master.txt"
```

Comment:
This won't be accessible on the grid, could you have this moved to `icarus_data` (by contacting @SFBayLaser, Tracy Usher) and verify that you can find the TextFile with this module